lib/utapi/utilities.js

```diff
@@ -57,23 +62,34 @@ function _listBucketMetrics(host,
     if (verbose) {
         logger.info('request headers', { headers: request._headers });
     }
-    request.write(JSON.stringify({ buckets, timeRange }));
+    const requestObj = { timeRange };
+    requestObj[metric] = metricType;
+    request.write(JSON.stringify(requestObj));
     request.end();
 }
 
-/**
- * This function is used as a binary to send a request to utapi server
- * to list bucket metrics
- *
- * @return {undefined}
- */
-export function listBucketMetrics() {
+export function listMetrics(metricType) {
     commander
         .version('0.0.1')
         .option('-a, --access-key <accessKey>', 'Access key id')
-        .option('-k, --secret-key <secretKey>', 'Secret access key')
-        .option('-b, --buckets <buckets>', 'Name of bucket(s)' +
-        'with a comma separator if more than one')
+        .option('-k, --secret-key <secretKey>', 'Secret access key');
+    // We want to continue support of previous bucket listing. Hence the ability
+    // to specify `metricType`. Remove `if` statement and
+    // bin/list_bucket_metrics.js when prior method of listing bucket metrics is
+    // no longer supported.
+    if (metricType === 'buckets') {
+        commander
+            .option('-b, --buckets <buckets>', 'Name of bucket(s) with ' +
+            'a comma separator if more than one');
+    } else {
+        commander
+            .option('-m, --metric <metric>', 'Metric type')
+            .option('--buckets <buckets>', 'Name of bucket(s) with a comma ' +
```

Comment:
I suppose we could keep the `-b` shortcut, but the problem is that then the shortcut for `--accounts` would intuitively seem to be `-a`. However, `-a` is already reserved for accessKey, so instead we decided to require the user to specify the metric with `-m` and then provide the extended arguments.